packages/svelte-query/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "./.svelte-kit/tsconfig.json",
```

Comment:
I't not sure this path exists? Shouldn't this extend our base config? E.g. like the react adapter:

https://github.com/TanStack/query/blob/f61b908014bf936ebd6f51758c746cd07eedf460/packages/react-query/tsconfig.json#L1-L13

note that we use `tsc -b` at the top level to create the `.d.ts` files for everything, so I think the configs need to match

Reply:
This is another sveltekit specific thing, the `./.svelte-kit` folder is generated by vite when the project runs or builds. Also, `svelte-package` doesn't need tsc to generate types - everything is handled by the single `build` script.